src/po/po_format.rs

```diff
@@ -379,7 +379,7 @@ fn convert_singular_po_message_to_fluent_message(
     }
 
     // Simply try to parse the unescaped content as a Fluent message value
-    let unescaped_msgstr = unescape_fluent_value(&msgstr);
+    let unescaped_msgstr = unescape_fluent_value(msgstr);
```

Comment:
💭  Question for my own curiosity: why is the `&` not necessary here? Doesn't `fn unescape_fluent_value(value: &str) -> String` expect a borrowed value, and as such aren't we expected to use `&msgstr` at call site? 🤔

Reply:
That's correct, but the `msgstr` is already a reference [here](https://docs.rs/polib/latest/polib/message/trait.MessageView.html#tymethod.msgstr):
```
let msgstr = message.msgstr()?;
```